spec/deprecation-equivalents/multicasting-deprecations-spec.ts

```diff
@@ -1,6 +1,6 @@
 /** @prettier */
 import { Observable, ConnectableObservable, connectable, of, AsyncSubject, BehaviorSubject, ReplaySubject, Subject, merge } from 'rxjs';
-import { connect, share, multicast, publish, publishReplay, publishBehavior, publishLast, refCount, repeat, retry } from 'rxjs/operators';
+import { connect, share, multicast, publish, publishReplay, publishBehavior, refCount, repeat, retry } from 'rxjs/operators';
```

Comment:
nit: It would be great if while we were changing these we stopped using the `rxjs/operators` exports and started using the `rxjs` exports.

Reply:
I have already researched this issue. I planned to make a separate PR. We can even include an eslint rule for this:

```json
"@typescript-eslint/no-restricted-imports": [
  "error",
  { "name": "rxjs/operators", "message": "Use 'rxjs'" }
],
```

Reply:
It's just that side changes add a lot of noise to the diff and make them hard to review.

Reply:
Makes sense to me. FYI: when I say "nit" I means I'm not going to block a merge on it. Just pointing something out. (As you may have noticed, if it was a "big deal" I'd have just added a commit to fix it).

Reply:
[Voila!](https://github.com/ReactiveX/rxjs/pull/7173)

Reply:
I'll probably add the eslint rule to the PR for #5078.